tests/api_data/test_data.py

```diff
@@ -47,15 +47,6 @@
     "project_name": "Project In prog and Res",
     "short_id": "INP",
     "flags": [
-        {
```

Comment:
removed as theres a new system of flagging in place, this will prob get a rework the db schema is done